sklearn/kernel_approximation.py

```diff
@@ -1,521 +1,638 @@
-"""
```

Comment:
actually probably an issue with line endings...see [this blog post](http://www.hanselman.com/blog/YoureJustAnotherCarriageReturnLineFeedInTheWall.aspx) and [this support post by github](https://help.github.com/articles/dealing-with-line-endings/)